Applications/KNN/jni/bitmap_helpers.cpp

```diff
@@ -109,8 +109,10 @@ uint8_t *read_bmp(const std::string &input_bmp_name, int *width, int *height,
   // Decode image, allocating tensor once the image size is known
   uint8_t *output = new uint8_t[abs(*height) * *width * *channels];
   const uint8_t *bmp_pixels = &img_bytes[header_size];
-  return decode_bmp(bmp_pixels, row_size, output, *width, abs(*height),
-                    *channels, top_down);
+  decode_bmp(bmp_pixels, row_size, output, *width, abs(*height), *channels,
```

Comment:
This might give error of "return value ignored". How about `output = decode_bmp(...)`?